chef/cookbooks/updater/recipes/default.rb

```diff
@@ -91,6 +116,12 @@
       end # block
     end # ruby_block
 
+    ["corosync", "pacemaker"].each do |s|
```

Comment:
It will be a no-op, because of the `Requires=corosync.service` dependency. But if you prefer to be safe (or support other distros which might be missing it), you can keep it, in which case I think there should be a comment explaining that it should be unnecessary and we're just playing safe.